src/main/java/games/strategy/net/ServerMessenger.java

```diff
@@ -613,7 +618,9 @@ public void run() {
 
   private TimerTask getUsernameUnmuteTask(final String username) {
     return createUnmuteTimerTask(
-        () -> (isLobby() && new MutedUsernameController().getUsernameUnmuteTime(username) == -1) || (isGame()),
+        () -> (isLobby()
+            && Instant.EPOCH.equals(new MutedUsernameController().getUsernameUnmuteTime(username).orElse(null)))
+            || isGame(),
```

Comment:
Could this be simplified by using `isUsernameMuted()` instead of `getUsernameUnmuteTime()`?

```java
return createUnmuteTimerTask(
    () -> (isLobby() && !new MutedUsernameController().isUsernameMuted(username)) || isGame(),
    () -> liveMutedUsernames.remove(username));
```